package.json

```diff
@@ -70,5 +71,8 @@
   },
   "workspaces": [
     "packages/*"
-  ]
+  ],
+  "resolutions": {
```

Comment:
This is require because

1. jest-cli uses babel-core: 6 version which is hoisted to the node_modules directory by using yarn workspaces which in turn hoists babel-register: 6
2. running babel-register with 6 version would break and report this following error which is deprecated in babel 7
```js
TypeError: /Users/vshanmugam/open-source/babili/gulpfile.babel.js: path.arrowFunctionToExpression is not a function
    at PluginPass.ArrowFunctionExpression (/Users/vshanmugam/open-source/babili/node_modules/babel-plugin-transform-es2015-arrow-functions/lib/index.js:10:14)

```